spec/condition_spec.rb

```diff
@@ -145,4 +145,19 @@
     condition = JSON.parse(condition)
     expect(@condition_evaluator.evaluate(condition)).to be true
   end
+
+  it 'should evaluate to true for evaluate when user attributes evaluate true' do
```

Comment:
Can you re-word this: `should evaluate to true when user attributes match the audience conditions`